packages/commonwealth/server/util/requirementsModule/requirementsSchema.json

```diff
@@ -0,0 +1,95 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "ContractSource": {
+      "type": "object",
+      "properties": {
+        "source_type": {
+          "type": "string",
+          "enum": ["erc20", "erc721"]
+        },
+        "chain_id": {
+          "type": "string"
```

Comment:
Should ETH chain IDs be strings, or numbers?

Reply:
ETH chain IDs are always strings in the DB

Reply:
Gotcha -- interesting because in theory they're numeric (ETH mainnet = 1, Goerli = 5, etc), so we might want to validate that the chain_ids are in fact numbers

Reply:
Oh I see. I thought Chain ID in the schema mapped to `Chain.id` in the DB (e.g. `ethereum`, `dydx`, etc.).

What you're referring to is the blockchain Chain ID.

Since the codebase refers to the DB model definition of `chain_id` 99% of the time (in threads, comments, topics, etc.), perhaps we should consider renaming the schema property to reduce confusion?

Something like `eth_chain_id`?

Reply:
Fine by me -- let's change it to evm_chain_id or eth_chain_id, whichever you think makes more sense.